src/pybamm/models/submodels/equivalent_circuit_elements/ocv_element.py

```diff
@@ -28,9 +28,11 @@ def get_coupled_variables(self, variables):
         current = variables["Current [A]"]
 
         ocv = variables["Open-circuit voltage [V]"]
+        ocv.print_name = "OCV"
         T_cell = variables["Cell temperature [degC]"]
 
         dUdT = self.param.dUdT(ocv, T_cell)
+        dUdT.print_name = r"\frac{du}{dt}"
```

Comment:
I would assume we would want this to be capitalized.
```suggestion
        dUdT.print_name = r"\frac{dU}{dT}"
```